packages/riverpod/lib/riverpod.dart

```diff
@@ -31,7 +31,9 @@ export 'src/framework.dart'
         handleFireImmediately,
         Create,
         Node,
-        ProviderElementProxy;
+        ProviderElementProxy,
+        OnError,
```

Comment:
I'm not fond of exporting them. The value is still, and it could cause name conflicts if someone already defines something with those names.

Reply:
I see. I'm hiding the new `typedef`s on these lines. Is it not correct?

Reply:
Oh I'm sorry, I got confused and thought it was a `show` not `hide`. Ignore me

LGTM!